database/query/eval_history.sql

```diff
@@ -93,3 +93,71 @@ INSERT INTO alert_events(
     $3,
     $4
 );
+
+-- name: ListEvaluationHistory :many
+SELECT s.id::uuid AS evaluation_id,
+       s.most_recent_evaluation as evaluated_at,
+       -- entity type
+       CASE WHEN ere.repository_id IS NOT NULL THEN 'repository'::entities
+            WHEN ere.pull_request_id IS NOT NULL THEN 'pull_request'::entities
+            WHEN ere.artifact_id IS NOT NULL THEN 'artifact'::entities
+       END AS entity_type,
+       -- entity id
+       CASE WHEN ere.repository_id IS NOT NULL THEN r.id
+            WHEN ere.pull_request_id IS NOT NULL THEN pr.id
+            WHEN ere.artifact_id IS NOT NULL THEN a.id
+       END AS entity_id,
+       -- entity name
+       CASE WHEN ere.repository_id IS NOT NULL THEN r.repo_name
+            WHEN ere.pull_request_id IS NOT NULL THEN pr.pr_number::text
```

Comment:
Again, shouldn't the name for PRs include the `repo_owner` and `repo_name` from the related `repositories` table?

Reply:
👍, see comment on `repo_name` above.